### PR TITLE
fixed login-redirect issue for authenticated user

### PR DESCRIPTION
**Pull request by AnishShah**
After a user is successfully logged in, and the user hits "https://bookie.io/login", then the user is not redirected to its recent page.
This pull request fixes this issue.

### CODE REVIEW

File: bookie/views/auth.py [hunk 1]
```diff
@@ -82,6 +83,12 @@ def login(request):
             message = "Failed login"
             AuthLog.login(login, False, password=password)
 
+    if isinstance(request.user, User):
```
Comment (mitechie) [inline bookie/views/auth.py]: I'd like this to be a bit smarter. What about not just checking there is a User, but that it's a current valid user in the system that is activated as a security measure?

File: bookie/views/auth.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 from bookie.models import IntegrityError
 from bookie.models.auth import UserMgr
 from bookie.models.auth import ActivationMgr
+from bookie.models.auth import User
```
Comment (mitechie) [inline bookie/views/auth.py]: I know it's not your doing but we've been trying to clean up the imports more into the format like:

```
from bookie.models.auth import (
    ActivationMgr,
    User,
    UserMgr,
)
```

File: bookie/tests/test_auth/test_api_base.py [hunk 3]
```diff
@@ -123,3 +123,28 @@ def test_login_null(self):
         self.assertTrue(
             'Failed login' in str(res),
             "Should have 'Failed login' in the resp: " + str(res))
+
+    def test_redirect_for_authenticated_user(self):
```
Comment (mitechie) [inline bookie/tests/test_auth/test_api_base.py:127]: given the updated sanity check below those should have matching tests to make sure we verify we force a login to invalid users.

**Comment by mitechie** [comment]
Thanks for this. I'd like to push on a little more awesomeness for this to land if that's ok.

File: bookie/tests/test_auth/test_api_base.py (hunk 3 above)
Comment (AnishShah) [inline bookie/tests/test_auth/test_api_base.py:127]: `test_login_url` test on Line26 already does that. That's why, I didn't add another test case.

**Comment by AnishShah** [comment]
@mitechie I have made the changes. 
If you want a separate test case for invalid user (which `test_login_url` already does) , then I'll do it.

**Comment by mitechie** [comment]
Thanks for the update!

**Pull request merged**